index.bs

```diff
@@ -547,17 +547,85 @@ navigator.getVRDisplays().then(function (displays) {
 </pre>
 </div>
 
+## VRDisplayEventReason ## {#interface-vrdisplayeventreason}
+
+<pre class="idl">
+interface VRDisplayEventReason {
+  readonly attribute unsigned short code;
+  const unsigned short VR_DISPLAY_NAVIGATION = 1;
+  const unsigned short VR_DISPLAY_MOUNTED = 2;
+  const unsigned short VR_DISPLAY_UNMOUNTED = 3;
+};
+</pre>
+
+### Attributes ### {#vrdisplayeventreason-attributes}
+
+<dfn attribute for="VRDisplayEventReason">code</dfn>
+The code describing the reason for this event, described below.
+
+### Reason Codes ### {#vrdisplayeventreason-codes}
```

Comment:
can we use strings instead of integer enums?

I like how the Permissions API handles `status`:

https://www.w3.org/TR/permissions/#idl-PermissionStatus-status
https://www.w3.org/TR/permissions/#h-examples